client/src/routes/signin/+page.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+  import { loginWithGoogle, loginWithMicrosoft } from "$lib/firebase/auth";
+</script>
+
+<div class="container">
+  <div class="google">
+    <button id="google-button" on:click={loginWithGoogle}>sign in with google</button>
+  </div>
+  <div class="microsoft">
+    <button id="microsoft-button" on:click={loginWithMicrosoft}>sign in with Microsoft</button>
+  </div>
+  <div class="email">
+    <button id="email-button">Sign in With Email</button>
+  </div>
+  <div id="input" />
+</div>
+
+<!-- <h1>{$id}</h1> -->
+<style>
+  .container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+    gap: 0px 0px;
+    grid-auto-flow: row;
+    grid-template-areas:
+      ". . . . . . . . ."
+      ". . . . . . . . ."
+      ". . . . . . . . ."
+      ". . . . google . . . ."
+      ". . . . microsoft . . . ."
+      ". . . . email . . . ."
+      ". . . . input . . . ."
+      ". . . . . . . . ."
+      ". . . . . . . . ."
+      ". . . . . . . . .";
```

Comment:
If you are trying to center you should consider using `place-content`
https://css-tricks.com/snippets/css/complete-guide-grid/#aa-place-content